src/tests/unit/tests/DetailsView/components/details-view-command-bar.test.tsx

```diff
@@ -98,31 +121,36 @@ describe('DetailsViewCommandBar', () => {
     });
 
     function testOnPivot(givenRenderExportAndStartOver: boolean): void {
-        renderExportAndStartOver = givenRenderExportAndStartOver;
-        const props = getProps();
+        const theHtml = 'this is the HTML';
+        let theDescription = null;
+        reportExportComponentProps = givenRenderExportAndStartOver
+            ? {
```

Comment:
I would create a variable to represent the reportExportComponentProps object here; makes it easier to parse and understand that way.

Reply:
Done